frontend/src/components/AdequacyCensusCharts/AdequacyCensusCharts.tsx

```diff
@@ -56,6 +57,47 @@ export let AdequacyCensusCharts = withStore('adequacies', 'method')<Props>(({ se
         )
       }
     </StatsBox>
+    <div>
+      <HorizontalBar
+        data={{
+          labels: censusGroups,
+          datasets: [{
+            backgroundColor: ADEQUACY_COLORS[AdequacyMode.ADEQUATE_15],
```

Comment:
Maybe use an array of [AdequacyMode.ADEQUATE_15, AdequacyMode.ADEQUATE_30, AdequacyMode.ADEQUATE_60, AdequacyMode.INADEQUATE] and then map on this